zulipterminal/model.py

```diff
@@ -778,9 +780,9 @@ def _handle_update_message_event(self, event: Event) -> None:
             # 'subject' is not present in update event if
             # the event didn't have a 'subject' update.
             if 'subject' in event:
+                new_subject = event['subject']
```

Comment:
I like this change to reduce dictionary lookups. :+1:

Reply:
This might make a marginal difference if there are multiple message ids in the event; this is mainly for readability and also to some degree having it fit on one line easily ;)